### ISSUE TITLE
Works with KRC5?

### ISSUE THREAD

**Issue by PicassoCT** [ImtsSrl/KUKAVARPROXY#8]
Not actually a issue, more a question.. sorry if wrong location.

**Comment by dddomodossola** [comment]
Never tested on it. If you get the chance to try please report.

**Comment by PicassoCT** [comment]
Will do.

**Comment by kwint** [comment]
@PicassoCT Where you ever able to test this? We currently have the same question

**Comment by PicassoCT** [comment]
> @PicassoCT Where you ever able to test this? We currently have the same question

No,  we ultimately went with a krc4 were this tool worked. I did however contact Kuka doing research for this. The tech-support representative assured us that the krc5 is mostly a hw overhaul and no major sw architectural changes have been performed.

 I can however not assure you whether a external tool relying on this will work. I will update this post if I we transfer to krc5.

**Comment by filhohao** [comment]
Hello guys! 
Is there any updates on this matter? Im about to order some new krc5 but am unsure due to this issue!

**Comment by filhohao** [comment]
Hello guys! 
So I received a KRC5 and I'm having issues with enabling KVP. 
Basically I cannot open a new port on the controller. 
Is it possible to change listening port for KVP?

**Comment by filhohao** [comment]
So some updates on this matter! the solution still works on KRC5 KSS 8.7.2
There are some difficulties on opening the door, but I was able to do it after a while.

**Comment by MrTorstein** [comment]
We have also tested this with the KRC5 Micro, and it works perfectly. I do not remember the KSS versions, but we just got a new micro last week and it worked after registering the cswsk32.ocx file

**Comment by PicassoCT** [comment]
@MrTorstein  Thank you for carrying the torch forward! This is what progress is down through the years.

PS: This Interface greatly benefited from burst movement communication, especially with interpolating small numerous movements. So stacking the movement communication up, send it, then execute it with a "interpreter".
